data/wpa_supplicant/wpa_supplicant_test.sh

```diff
@@ -14,6 +14,7 @@ function cleanup() {
 		kill $hostapd_pid
 	fi
 	modprobe -r mac80211_hwsim
+        cat hostapd.log
```

Comment:
Do we need this for the test?

Reply:
Not really, I  have just added line to get the results after the hostapd run.

Reply:
Just as suggestion: One could think about using [`upload_logs`](http://open.qa/api/testapi/#_upload_logs) in the post-run and post-fail hooks to upload this log as an openQA asset. See e.g. [docker](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/containers/docker.pm#L72) if you feel motivated :slightly_smiling_face: